common/util.c

```diff
@@ -54,3 +55,23 @@ float parse_float(const char *value) {
 	}
 	return flt;
 }
+
+
+const char *sway_wl_output_subpixel_to_string(enum wl_output_subpixel subpixel) {
+	switch (subpixel) {
+		case WL_OUTPUT_SUBPIXEL_UNKNOWN:
+			return "unknown";
+		case WL_OUTPUT_SUBPIXEL_NONE:
+			return "none";
+		case WL_OUTPUT_SUBPIXEL_HORIZONTAL_RGB:
+			return "rgb";
+		case WL_OUTPUT_SUBPIXEL_HORIZONTAL_BGR:
+			return "bgr";
+		case WL_OUTPUT_SUBPIXEL_VERTICAL_RGB:
+			return "vrgb";
+		case WL_OUTPUT_SUBPIXEL_VERTICAL_BGR:
+			return "vbgr";
+		default:
```

Comment:
Please don't add `default` cases, the compiler won't tell you where your code needs an update if a new enum member is added. Instead, add `assert(false)` after the `switch`.

Reply:
I see one `assert(false)` in the codebase, so I used sway_assert() with an error message instead. Anyway, default case removed.